src/libraries/Common/tests/System/Collections/ICollection.Generic.Tests.cs

```diff
@@ -334,6 +336,46 @@ public void ICollection_Generic_Clear_Repeatedly(int count)
             }
         }
 
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsPreciseGcSupported))]
```

Comment:
are we covered for TrimExcess() (where available)?

Reply:
On all of Stack, Queue, List, HashSet, Dictionary

Reply:
I don't think TrimExcess is relevant for this.  It's shrinking the size of the array after items have already been removed; it's the removal that's impacted, not the subsequent shrinking.

Reply:
Yes I don't know what o was thinking 😊